samcli/local/docker/lambda_container.py

```diff
@@ -96,16 +96,21 @@ def _get_exposed_ports(debug_options):
         bind to same port both inside and outside the container ie. Runtime process is started in debug mode with
         at given port inside the container and exposed to the host machine at the same port
 
-        :param int debug_port: Optional, integer value of debug port
+        :param list(int) debug_port: Optional, List of integer values of debug ports
         :return dict: Dictionary containing port binding information. None, if debug_port was not given
         """
         if not debug_options:
             return None
 
-        return {
-            # container port : host port
-            debug_options.debug_port: debug_options.debug_port
-        }
+        if not debug_options.debug_port:
+            return None
+
+        # container port : host port
+        ports_map = {}
+        for port in debug_options.debug_port:
```

Comment:
Can you update the docs for `debug_options.debug_port` to reflect changes to the type?